### DISCUSSION TITLE
GitHub Discussions Open

### DISCUSSION THREAD

**Discussion by jeremyevans** [jeremyevans/roda#218] [General]
Previously, Roda used the ruby-roda Google Group as a discussion forum/mailing list. That will continue to be available for people who do not want to use GitHub Discussions, but GitHub Discussions is now the preferred discussion forum for Roda. Feel free to ask any questions regarding Roda here. Bugs in Roda should still be reported via GitHub Issues.

**Comment by jaredcwhite** [comment]
Ooo, exciting. I greatly prefer using GH to Google Groups so this is a welcome change. Thanks Jeremy!
